test_autograph.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+## for testing autograph mar signing from signingscript
+
+mkdir -p artifact_dir/
+mkdir -p work_dir/cot/upstream-task-id1/
+
+test -f firefox-mozilla-central-58.0a1-linux-x86_64-en-US-20171001220301-20171002220204.partial.mar || (
+    wget http://ftp.mozilla.org/pub/firefox/nightly/latest-mozilla-central/firefox-mozilla-central-58.0a1-linux-x86_64-en-US-20171001220301-20171002220204.partial.mar \
+	&& \
+	cp firefox-mozilla-central-58.0a1-linux-x86_64-en-US-20171001220301-20171002220204.partial.mar work_dir/cot/upstream-task-id1/
+)
+test -f firefox-mozilla-central-58.0a1-linux-x86_64-en-US-20171001220301-20171002100134.partial.mar || (
+    wget http://ftp.mozilla.org/pub/firefox/nightly/latest-mozilla-central/firefox-mozilla-central-58.0a1-linux-x86_64-en-US-20171001220301-20171002100134.partial.mar \
+	&& \
+	cp firefox-mozilla-central-58.0a1-linux-x86_64-en-US-20171001220301-20171002100134.partial.mar work_dir/cot/upstream-task-id1/
+)
+
+# not using the signingserver so any example_server_config.json will do
+cat <<EOF > autograph_test_server_config.json
+{
+    "project:releng:signing:cert:dep-signing": [
+        ["https://autograph-hsm.dev.mozaws.net", "alice", "fs5wgcer9qj819kfptdlp8gm227ewxnzvsuj9ztycsx08hfhzu", ["autograph_mar384"], "autograph"]
```

Comment:
What's the risk of letting an attacker use these (public) credentials?

Reply:
They're only linked to test/dummy keys and already public in the example autograph config: https://github.com/mozilla-services/autograph/blob/d6a77dd438ca1fa453de46426db48cbe722e8d6f/autograph.yaml#L330-L331

Reply:
Awesome!